bin/topological_inventory-orchestrator

```diff
@@ -9,6 +9,8 @@ $:.push File.expand_path("../../lib", __FILE__)
 def parse_args
   require 'optimist'
   opts = Optimist.options do
+    opt :collector_image_tag, "Image tag to use for the collector images, e.g. latest, stable", :type => :string,
+        :default => ENV["COLLECTOR_IMAGE_TAG"], :required => ENV["COLLECTOR_IMAGE_TAG"].nil?
```

Comment:
Based on https://github.com/RedHatInsights/e2e-deploy/pull/349/files this should be `IMAGE_TAG`